stdlib/public/SwiftRemoteMirror/SwiftRemoteMirror.cpp

```diff
@@ -224,143 +248,157 @@ ReflectionSection<Iterator> reflectionSectionFromLocalAndRemote(
 void
 swift_reflection_addReflectionInfo(SwiftReflectionContextRef ContextRef,
                                    swift_reflection_info_t Info) {
-  auto Context = ContextRef->nativeContext;
-
-  // The `offset` fields must be zero.
-  if (Info.field.offset != 0
-      || Info.associated_types.offset != 0
-      || Info.builtin_types.offset != 0
-      || Info.capture.offset != 0
-      || Info.type_references.offset != 0
-      || Info.reflection_strings.offset != 0) {
-    std::cerr << "reserved field in swift_reflection_info_t is not zero\n";
-    abort();
-  }
-
-  ReflectionInfo ContextInfo{
-      sectionFromInfo<FieldDescriptorIterator>(Info, Info.field),
-      sectionFromInfo<AssociatedTypeIterator>(Info, Info.associated_types),
-      sectionFromInfo<BuiltinTypeDescriptorIterator>(Info, Info.builtin_types),
-      sectionFromInfo<CaptureDescriptorIterator>(Info, Info.capture),
-      sectionFromInfo<const void *>(Info, Info.type_references),
-      sectionFromInfo<const void *>(Info, Info.reflection_strings),
-      ReflectionSection<const void *>(nullptr, 0),
-      ReflectionSection<MultiPayloadEnumDescriptorIterator>(0, 0),
-      {}};
+  ContextRef->withContext<void>([&](auto *Context) {
```

Comment:
I sure wish GitHub could be smarter about displaying diffs when the only thing that changed is indentation.

Reply:
https://github.com/apple/swift/pull/71168/files?w=1 :)

Reply:
Too funny! And useful!